### DISCUSSION TITLE
Discusiones de la Malla Interactiva

### DISCUSSION THREAD

**Discussion by BooterMan98** [BooterMan98/malla-interactiva#22] [General]
# Hola!

Espero que les hayan gustado mucho las adiciones que les he dado al proyecto. La idea que tengo para este lugar es para que publiquen preguntas, dudas y discutamos ideas sobre la malla. Solo les pido que mantengan un ambiente sano :)

xd

**Comment by amlaihak** [comment]
Hola!
Estaba buscando mi malla de carrera pero resulta que la pagina está caída ¿Hay alguna posibilidad de que labcomp vuelva o la pagina murió definitivamente?

**Reply by BooterMan98** [reply]
Hola!

No, **no ha muerto**. Labcomp este año renovó a todos sus ayudantes y entonces no tienen a alguien con tiempo que les pueda ayudar y dejarlos preparados para cualquier circunstancia. **Solo hay que esperar un poco** mientras aprenden sobre la marcha. Si estas muy urgido en el readme hay un link a una versión alojada en GitHub. :)

**Comment by maripidays** [comment]
hola! una amiga que asiste a la usm me mostro lo bacan de la malla interactiva y pensé que seria super útil para muchos, pero como estaban solo las carreras de la usm pensé que podría agregar la mía en generar malla, pero creo que no se puede o quizá yo no supe hacerlo jeje, entonces quería saber si es que era posible así como agregar una malla de otra universidad para poder usarlo o solo se limita a las carreras de la usm, agregue todos mis ramos con créditos y todo y cuando puse generar solo me salieron los del primer semestre no supe que mas hacer, aunque super buena la idea! es super útil!

**Reply by BooterMan98** [reply]
Hola!

Se agradece el interés en usar la malla, pero por el momento no se aceptan mallas provenientes de otras universidades. Si se aceptan, tendría que estar al tanto de actualizaciones, corrección de errores, solicitud de cambios de cada carrera disponible y eso sería mucho para 1 persona.

De todas formas me gustaría poder aceptarlas en algún momento, pero se requiere tiempo y planificación que no he puesto al proyecto.

Ahora, la opción de generar malla es capaz de generar una que fuera de otra universidad. Si aún tienes problemas podrías crear una [Issue](https://github.com/BooterMan98/malla-interactiva/issues), adjuntando una foto de la malla antes y después de generarla para intentar ayudarte. :P

**Comment by Manu-pardo** [comment]
Hola! Será factible incorporar la malla de la nueva carrera de ingeniería civil física?, la malla mas completa está en la pagina del departamento de física.

Gracias!

**Comment by Cubozoa1818** [comment]
Hola! Quería preguntar si será posible añadir la carrera de ingeniería mecánica industrial, es una carrera que la imparte el departamento de mecánica en casa central, sería de mucha utilidad que la añadan en la pagina debido a que no hay mucha info de ella en internet.

Gracias!

**Comment by kike-rc** [comment]
Hola! generé mi propia malla y corregí los créditos SCT de algunos ramos. Sin embargo, al sumar los créditos de los ramos corregidos, en lugar de obtener el resultado de la suma, los valores se colocan uno junto al otro, formando un número largo. ¿Cómo podría solucionarlo?

**Reply by BooterMan98** [reply]
Hola! no entiendo el problema, a que te refieres con la suma? podrías adjuntar una foto?

**Reply by kike-rc** [reply]
![Captura de pantalla 2024-12-15 215531](https://github.com/user-attachments/assets/8f881edb-1b18-4291-b9a3-b9d2779c269f)

**Comment by Sebanfe** [comment]
Hola buenas, estoy intentado hacer una malla de otra universidad, dado que la aplicacion es muy buena, pero al agregar ramos estos se bugean de hartas formas, al agregar asignaturas y luego tratar de colocarlas, no se dejan seleccionar y al tratar de cambiar de semestre se empiezan a agregar un ramo de forma repetitiva.
![foto](https://github.com/user-attachments/assets/81009853-8e51-401c-af92-29463dc178d5)

**Comment by Dinarpb** [comment]
Hola! Estuve intentando crear mi propia malla, pero al momento de generar la malla únicamente me genera hasta el semestre 8 y tiene 10 :(
Además al momento de querer editarlo, me deselecciona todos los cursos seleccionados después del semestre 2. Además al momento de intentar seleccionar un curso en la malla ya creada no me deja. Me encanta la propuesta pero no se como solucionar eso :(

**Comment by NeuraAdm** [comment]
Hola, me gusta demasiado el proyecto, pero seria bueno que lo abrieran con posibilidad de otras carreras de otras partes que no sea de Chile, para que asi funcione correctamente y se pueda ver todas las buenas funcionalidades que tiene este aplicativo.

**Reply by BooterMan98** [reply]
Hola!

Estoy **completamente de acuerdo**! Sinceramente eso no se ha dado porque no le dado tiempo al proyecto, para abrirme a otras Universidades de Chile (Solo muestra mallas de la [USM](usm.cl)) y luego de otras partes hay que hacer unos cambios. Eso es reescribir parte o todo el proyecto para que sea más escalable, facilitar la colaboración y corregir alguno de los bugs como los que se presentan más arriba.  
Ya que el proyecto esta teniendo otro momento viral (gracias a TikTok me imagino) estoy considerando dedicarle un poco más de tiempo.

**Comment by iolivarec** [comment]
Hola!, ex-ayudante del labcomp acá.

Como todavía tengo acceso a la administración del dominio labcomp.cl y el antiguo coordinador no lo actualizaba desde enero de 2024, apliqué los cambios más recientes (vigentes desde el 15 de diciembre de 2025) en mallas.labcomp.cl con un fork mío. Muchas gracias por mantener el proyecto al día con las nuevas mallas curriculares!

**Comment by BenjaminPinoPyton** [comment]
Hola me gusta caleta el proyecto, me puse a hacer la malla nueva de civil industrial, voy en el s5, tengo que ponerle bueno nomas de aqui a un par de semanas mando el doc con todo.
